persist/schema/types.go

```diff
@@ -92,4 +92,5 @@ type LogMetadata struct {
 	ID        []byte
 	Namespace []byte
 	Shard     uint32
+	Tags      []byte
```

Comment:
nit: Should we call this `EncodedTags`?  I don't feel strongly, however it is what it's called in the `IndexEntry`.